test/e2e/move.update.stc.test.js

```diff
@@ -40,3 +46,154 @@ test.meta('hasDocument', 'true')(
     ])
   }
 )
+
+test('User should be able to cancel move', async () => {
+  await checkCancelLink()
+})
+
+// fixture(
```

Comment:
Rogue comments?

Reply:
@teneightfive I needed to see how to filter by hospital, so now I know how to do that, I've uncommented these lines.